src/js/constants.js

```diff
@@ -61,6 +61,37 @@ export const MediaStateReceiverAttributes = {
   MEDIA_CONTROLLER: 'mediacontroller',
 };
 
+export const MediaUIProps = {
```

Comment:
Should we consider deriving the various other dictionary objects from this for easier maintainability? i.e. make`MediaUIAttributes` and `MediaStateChangeEvents` just a `reduce()` translation of this?

Reply:
I like that 👍

Reply:
we can apply ad hoc rules to do things like omit, but I'd rather treat them as exceptions in the codebase. Should be easier to maintain.